src/domain/schemas/document-schemas.js

```diff
@@ -53,7 +53,7 @@ export const hardDeleteDocumentBodySchema = joi.object({
   documentKey: idOrKeySchema.required()
 });
 
-const sectionDBSchema = joi.object({
+export const sectionDBSchema = joi.object({
```

Comment:
We probably should not use this, but for now it's okay. It has some properties which I think are debatable if we need such as "deleteBecause"

Reply:
True, that's a good question. See also my other comment regarding this.